services/supervisor/src/schema.rs

```diff
@@ -80,4 +85,22 @@ impl Schema<Prefixed<'_, &Fork>> {
         let new_configuration_number = self.configuration_number.get().unwrap_or(0) + 1;
         self.configuration_number.set(new_configuration_number);
     }
+
+    /// Assign unique identifier for an instance.
+    pub(crate) fn assign_instance_id(&mut self) -> InstanceId {
+        let id = self
+            .vacant_instance_id
+            .get()
+            .unwrap_or(MAX_BUILTIN_INSTANCE_ID);
```

Comment:
What about to rename into `FIRST_DEPLOYED_INSTANCE_ID` ? As I can see builtin instance can't get id 1024.

Reply:
It'll get removed soon anyway, so I don't think it matters (see ECR-3851) :)